packages/css/src/util/fetch/StatusDependant.ts

```diff
@@ -0,0 +1,7 @@
+
+/**
+ * Any object of which a status can be changed.
+ */
+export interface StatusDependant<T> {
+  changeStatus(status: T): Promise<void>, 
```

Comment:
Isn't that almost any object? 😅 Very subjective but name feels weird to me. I'm thinking more of something like `setState` and as interface name `StateMachine` or something similar?

Reply:
Yeah, I know, terrible name 😆  I was wondering for the CSS: do you prefer we just implement this specifically for the Fetchers, or do we keep this and change the name to something better?

Reply:
I'm ok with a generic interface as not all fetchers will need this, and there might be other components in the future that also depend on something similar. But I would prefer a better name 😀